### ISSUE TITLE
MovieDrawable does not scale at all when ImageView's scaleType is fitXY

### ISSUE THREAD

**Issue by littledot** [coil-kt/coil#206]
**Describe the bug**
Consider the following layout
```xml
<LinearLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:orientation="vertical"
    >

    <ImageView
        android:id="@+id/imgView"
        android:layout_width="500px"
        android:layout_height="500px"
        android:background="#0FF"
        android:scaleType="fitXY"
        />

    <ImageView
        android:id="@+id/imgView2"
        android:layout_width="500px"
        android:layout_height="500px"
        android:background="#F0F"
        />

</LinearLayout>
```
which is loaded with 2 gifs
```kotlin
class MainActivity : AppCompatActivity() {
    
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.coil_gif_scale)
        
        val src =
            "https://media1.giphy.com/media/mTANPe0FgBuo/giphy.gif?cid=4d1e4f2998945bed1522c2608e7db022535fa1bfbd50ad4b&rid=giphy.gif"
        Coil.loadAny(this, src) {
            target(imgView)
            decoder(GifDecoder())
        }
        
        Coil.loadAny(this, src) {
            target(imgView2)
            decoder(GifDecoder())
        }
    }
}
```
will yield the following output:

![](https://i.imgur.com/l8dYSTc.png)

Is it expected that specifying `scaleType:fitXY` in the ImageView to cause Coil to not perform any scaling?

I was hoping Coil to at least use `Scale.FIT`.

**Expected behavior**

**To Reproduce**
Run the project: https://gitlab.com/littledot/android-bug-report

**Logs/Screenshots**
If applicable, add logs or screenshots to help explain your problem.

**Version**
What library version are you using? Also does this occur on a specific API level or Android device.
Coil 0.8.0

**Comment by colinrtwhite** [comment]
Thanks for the report + sample project. This should be fixed by https://github.com/coil-kt/coil/pull/202. Can you try the [`0.9.0-SNAPSHOT`](https://coil-kt.github.io/coil/faq/#how-do-i-get-development-snapshots) and let me know if that fixes the issue? The stable `0.9.0` release should be out later this week.

**Comment by littledot** [comment]
Awesome, this is indeed fixed in 0.9.0
